lib/ansible/executor/interpreter_discovery.py

```diff
@@ -150,6 +151,8 @@ def discover_interpreter(action, interpreter_name, discovery_mode, task_vars):
         return platform_interpreter
     except NotImplementedError as ex:
         display.vvv(msg=u'Python interpreter discovery fallback ({0})'.format(to_text(ex)), host=host)
+    except AnsibleError as ex:
+        raise ex
```

Comment:
you can remove `is ex` and `ex`, no need to create the variable, by default the existing exception will be reraised